source/_integrations/nest.markdown

```diff
@@ -26,6 +26,12 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
+<div class='note warning'>
+<p>The instructions on this page do not work for new credentials due to a change in Google's desktop authentication. See <a href="https://github.com/home-assistant/core/issues/67662">this GitHub issue</a> for details.</p>
+
+<p>Existing credentials will continue to work until October 2022.</p>
+</div>
```

Comment:
This should be an alert instead?

<https://github.com/home-assistant/alerts.home-assistant.io>

?

Reply:
Going to resolve this as this PR is not around alerts, but in another iteration that seems like a good idea

Reply:
I'll think about the alert more, i think that should be on me and not you.